docs/contracts/node-operators-registry.md

```diff
@@ -3,7 +3,32 @@
 - [Source Code](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.4.24/nos/NodeOperatorsRegistry.sol)
 - [Deployed Contract](https://etherscan.io/address/0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5)
```

Comment:
I bet it would be great to have a scheme describing the following terms and agreements:
- total
- vetted
- used (active)
- exited

and their derivatives:
- depositable
- unused
- submitted
- whatever